src/main/java/codeu/controller/BotHandler.java

```diff
@@ -0,0 +1,9 @@
+package codeu.controller;
+
+import codeu.model.data.BotRequest;
+import codeu.model.data.BotStructure;
+
+public interface BotHandler {
+  public String handler(BotRequest request);
```

Comment:
What should `handler` do? Describe it's usage and what implementers should be providing